etc/inc/disable-programs.inc

```diff
@@ -1280,6 +1280,8 @@ blacklist ${RUNUSER}/*firefox*
 blacklist ${RUNUSER}/akonadi
 blacklist ${RUNUSER}/i3
 blacklist ${RUNUSER}/psd/*firefox*
+blacklist ${RUNUSER}/*floorp*
+blacklist ${RUNUSER}/psd/*floorp*
```

Comment:
The entries in this file should be sorted, see:

* <https://github.com/netblue30/firejail/actions/runs/14016012927/job/39246281718?pr=6683>
* `ci/check/profiles/sort-disable-programs.sh`